challenge.rb

```diff
@@ -1,26 +1,32 @@
 
 def capitalize_each_string(input)
   #implement your solution here
+  return input.map { |e| e.capitalize }
```

Comment:
omit `return` statements when possible.  (This is a standard ruby styling thing.)